packages/story-utils/src/__tests__/generateSnapshot.test.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import generateSnapshot from '../generateSnapshot';
+import * as storyFileExports from './__mock_stories__/foo.stories';
```

Comment:
For naming, I think `snapshotTestStoryFile` and for the params `storyFileExports`. I'm still kind of partial to `anthology` despite it not being standard - seems like I wasn't the first to come up with that term - https://github.com/storybookjs/storybook/issues/686#issuecomment-337734028 though I didn't find other examples.